src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Queries/SearchProductQueryHandler.cs

```diff
@@ -103,7 +130,10 @@ public virtual async Task<SearchProductResponse> Handle(SearchProductQuery reque
                 Currency = currency,
                 Store = store,
                 Results = products,
-                Facets = resultAggregations?.Select(x => _mapper.Map<FacetResult>(x)).ToList(),
+                Facets = resultAggregations?.Select(x => _mapper.Map<FacetResult>(x, options =>

```

Comment:
The better solution will be to filter the resulting "synthetic" facets for mutilating properties according to the passed culture in this line. All code above can be removed.
e.g
`my-cool-property, my-cool-property_en_us, "my-cool-property_de_de` -> left only single value `my-cool-property` with values from `my-cool-property_en_us`